swagger.yaml

```diff
@@ -1,7 +1,7 @@
 swagger: '2.0'
 info:
   title: Identify API
-  description: Get an ONID username of a person.
+  description: Get information about a person using their 9 digit OSU ID.
```

Comment:
Can you remove `using their 9 digit OSU ID`?  They can also search by OSUUID or PROX ID.